lib/parameters.py

```diff
@@ -612,17 +612,17 @@ def gen_vp9_cqp_variants(spec):
     variants = params.get("variants", dict()).get("cqp", None)
 
     if variants is None:
-      keys = ["ipmode", "qp", "quality", "refmode", "looplvl", "loopshp"]
-      product = itertools.product([0, 1], [14, 28], [1, 4, 7], [0, 1], [0, 16, 63], [0, 4, 7])
+      keys = ["ipmode", "qp", "quality", "slices", "refmode", "looplvl", "loopshp"]
+      product = itertools.product([0, 1], [14, 28], [1, 4, 7], [1, 3], [0, 1], [0, 16, 63], [0, 4, 7])
       variants = [dict(zip(keys, vals)) for vals in product]
 
     for variant in variants:
       yield [
-        case, variant["ipmode"], variant["qp"], variant["quality"],
+        case, variant["ipmode"], variant["qp"], variant["quality"], variant["slices"],
```

Comment:
Please make slices optional with a default for user config.  We already have some cases defined without slices.

`variant.get("slices", 1)`

Do same for the other vp9 generators, please.

Reply:
This is why the pre-check failed

Reply:
updated